cmd/ipfs/main.go

```diff
@@ -103,9 +103,13 @@ func main() {
 	}
 
 	// Handle `ipfs help'
-	if len(os.Args) == 2 && os.Args[1] == "help" {
-		printHelp(false, os.Stdout)
-		os.Exit(0)
+	if len(os.Args) == 2 {
+		if os.Args[1] == "help" {
+			printHelp(false, os.Stdout)
+			os.Exit(0)
+		} else if os.Args[1] == "--version" {
+			os.Args[1] = "version"
+		}
```

Comment:
- this captures most of the times people will write `--version`
- but it may lead to weird behavior `(ipfs --timeout=10s --version` wont work).
- and it doesn't work on the API (making it only a CLI thing, which could cause confusion).
- I'm fine with it, but noting it's hacky